code/procs/mobprocs/chatprocs.dm

```diff
@@ -293,10 +293,10 @@
 	if (!message)
 		return
 
-	if (istype(src, /mob/living/critter/changeling/handspider))
+	if (isvampire(src))
 		name = src.real_name
 		alt_name = " (VAMPIRE)"
-	else if (istype(src, /mob/living/critter/changeling/eyespider))
+	else if (isvampiriczombie(src))
 		name = src.real_name
 		alt_name = " (GHOUL)"
```

Comment:
@Kyle2143 !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
May I suggest having it be something that Engineering is more equipped to handle rather than the chaplain? Maybe it starts draining power ala power sinks or it starts taking in gas that then cannot be regained. And to fight it off, engineering has to oil it up, then replace the superconductor, and finally use the lubricantion to either clean it out or add some space lube to have the fans run better. This calms down the engine and returns it to normal.